### ISSUE TITLE
Stand-Alone ArcGIS Server Authentication Fails

### ISSUE THREAD

**Issue by DCHuber** [Esri/arcgis-python-api#752]
**Describe the bug**
Using v1.8.2, connection to a stand-alone ArcGIS Server fails to authenticate.   Same code in v1.7.0 works fine

**To Reproduce**
```python
from arcgis.gis import server
user = '<admin user>'
pswd = '<password>'
server_base_url = "https://<server>/<webadaptor>"
gis_server = server.Server(url=f"{server_base_url}/admin",
                   token_url=f"{server_base_url}/tokens/generateToken",
                   username=user,
                   password=pswd)
service = gis_server.content.get('<service name>', folder='<folder>')
service.service.status
```
error:
```python
{'status': 'error', 'messages': ["Unauthorized access. Token not found. You can generate a token using the 'generateToken' operation."], 'code': 499}
```
**Screenshots**

**Expected behavior**
Same code in v1.7.0 returns:
```python
{'configuredState': 'STARTED', 'realTimeState': 'STARTED'}
```
**Platform (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
 - Python API Version: 1.8.2

**Additional context**
conda environment list:
#
arcgis                    1.8.2                 py36_1275    esri
arcgispro                 2.5                           0    esri
asn1crypto                1.3.0                    py36_0
atomicwrites              1.4.0                      py_0
attrs                     19.3.0                     py_0
backcall                  0.2.0                      py_0
beautifulsoup4            4.9.1                     <pip>
blas                      1.0                         mkl
bleach                    3.1.5                      py_0
blinker                   1.4                      py36_0
brotlipy                  0.7.0           py36he774522_1000
ca-certificates           2020.6.24                     0
certifi                   2020.6.20                py36_0
cffi                      1.14.0           py36h7a1dbc1_0
cftime                    1.0.0b1                  py36_0    esri
chardet                   3.0.4                 py36_1003
colorama                  0.4.3                      py_0
cryptography              2.8                      py36_0    esri
cycler                    0.10.0           py36h009560c_0
decorator                 4.4.2                      py_0
defusedxml                0.6.0                      py_0
despatch                  0.1.0                    py36_0    esri
entrypoints               0.3                      py36_0
et_xmlfile                1.0.1                   py_1001
fastcache                 1.1.0            py36he774522_0
freetype                  2.9.1                    vc14_0  [vc14]  esri
future                    0.18.2                   py36_0    esri
gdal                      2.3.3               arcgispro_1  [arcgispro]  esri
h5py                      2.9.0           py36_arcgispro_2  [arcgispro]  esri
html5lib                  1.1                        py_0
icc_rt                    2019.0.5            arcgispro_0  [arcgispro]  esri
idna                      2.10                       py_0
importlib-metadata        1.7.0                    py36_0
importlib_metadata        1.7.0                         0
intel-openmp              2020.0            arcgispro_166  [arcgispro]  esri
ipykernel                 5.3.3            py36h5ca1d4c_0
ipython                   7.16.1           py36h5ca1d4c_0
ipython_genutils          0.2.0                    py36_0
ipywidgets                7.5.1                      py_0
jdcal                     1.4.1                      py_0
jedi                      0.16.0                   py36_0    esri
jinja2                    2.11.2                     py_0
json5                     0.9.4                    py36_0    esri
jsonschema                3.2.0                    py36_0
jupyter_client            5.3.1                    py36_0    esri
jupyter_console           6.1.0                    py36_0    esri
jupyter_core              4.6.3                    py36_0    esri
jupyterlab                2.0.1                    py36_0    esri
jupyterlab_server         1.2.0                      py_0
keyring                   21.2.0                   py36_0    esri
kiwisolver                1.2.0            py36h74a9793_0
lerc                      2.2                        py_0    esri
libpng                    1.6.37               h2a8f88b_0
libsodium                 1.0.18                        1    esri
markupsafe                1.1.1            py36he774522_0
matplotlib                3.1.1                   py36_0e  [arcgispro]  esri
mistune                   0.8.4            py36he774522_0
mkl                       2019.4            arcgispro_245  [arcgispro]  esri
mkl-service               2.3.0            py36hb782905_0
mkl_fft                   1.1.0            py36h45dec08_0
mkl_random                1.1.0            py36h675688f_0
more-itertools            8.4.0                      py_0
mpmath                    1.1.0                    py36_0
nbconvert                 5.6.1                    py36_0    esri
nbformat                  5.0.7                    py36_0    esri
netcdf4                   1.5.1.2         py36_arcgispro_1  [arcgispro]  esri
nose                      1.3.7                    py36_2
notebook                  6.0.1                    py36_0
ntlm-auth                 1.5.0                     <pip>
ntlm-auth                 1.4.0                      py_0    esri
numexpr                   2.7.0            py36h6288b17_0
numpy                     1.16.6           py36h5510c5b_0
numpy-base                1.16.6           py36hc3f5095_0
oauthlib                  3.1.0                      py_0
openpyxl                  3.0.4                      py_0
openssl                   1.1.1g               he774522_0
packaging                 20.4                       py_0
pandas                    1.0.5            py36h47e9c7a_0
pandocfilters             1.4.2                    py36_1
parso                     0.7.0                      py_0
pickleshare               0.7.5                    py36_0
pip                       20.1.1                   py36_1
pluggy                    0.13.1                   py36_0
pro_notebook_integration  2.5                      py36_9    esri
prometheus_client         0.7.1                    py36_0    esri
prompt-toolkit            3.0.5                      py_0
prompt_toolkit            3.0.5                      py_0    esri
psutil                    5.7.0            py36he774522_0
py                        1.9.0                      py_0
pybind11                  2.3.0                         1    esri
pycparser                 2.20                       py_2
pygments                  2.6.1                    py36_0    esri
pyjwt                     1.7.1                    py36_0
pyopenssl                 19.1.0                     py_1
pyparsing                 2.4.7                      py_0
pypdf2                    1.26.0                     py_2    esri
pyrsistent                0.16.0           py36he774522_0
pyshp                     2.1.0                      py_0
pysocks                   1.7.1                    py36_0
pytest                    5.4.1                    py36_0    esri
python                    3.6.10                        1    esri
python-dateutil           2.8.1                      py_0
pytz                      2019.3                   py36_0    esri
pywin32-ctypes            0.2.0            py36he119be9_0    esri
pywin32-security          227                      py36_1    esri
pywinpty                  0.5.7                    py36_0    esri
pyzmq                     19.0.0                   py36_0    esri
requests                  2.24.0                     py_0
requests-kerberos         0.12.0                        0    esri
requests-kerberos         0.12.0                    <pip>
requests-oauthlib         1.3.0                      py_0
requests-toolbelt         0.9.1                      py_0
requests_ntlm             1.1.0                      py_0    esri
scipy                     1.5.0            py36h9439919_0
send2trash                1.5.0                    py36_0
setuptools                49.2.0                   py36_0
simplegeneric             0.8.1                    py36_2
six                       1.15.0                     py_0
soupsieve                 2.0.1                     <pip>
sqlite                    3.32.3               h2a8f88b_0
sympy                     1.5.1                    py36_0    esri
terminado                 0.8.3                    py36_0
testpath                  0.4.4                      py_0
tornado                   6.0.4            py36he774522_1
traitlets                 4.3.3                    py36_0
urllib3                   1.25.9                     py_0
vc                        14.1                 h0510ff6_4
vs2015_runtime            14.16.27012          hf0eaf9b_0    esri
wcwidth                   0.2.5                      py_0
webencodings              0.5.1                    py36_1
wheel                     0.34.2                   py36_0
widgetsnbextension        3.5.1                    py36_0
win_inet_pton             1.1.0                    py36_0
wincertstore              0.2              py36h7fe50ca_0
winkerberos               0.7.0                    py36_1
winpty                    0.4.3                         4
x86cpu                    0.4                      py36_1    esri
xeus                      0.23.2                        3    esri
xlrd                      1.2.0                      py_0
xlwt                      1.3.0            py36h1a4751e_0
zeromq                    4.3.2                         1    esri
zipp                      3.1.0                      py_0
zlib                      1.2.11               h62dcd97_4

**Comment by bixb0012** [comment]
Like the same root cause as #748 , which they have stated will be addressed in 1.8.3.

**Comment by achapkowski** [comment]
This issue has been resolve, please update the ArcGIS API for Python to the latest and test again.
